packages/react-combobox/src/components/OptionGroup/useOptionGroupStyles.ts

```diff
@@ -1,25 +1,49 @@
+import { tokens } from '@fluentui/react-theme';
 import { SlotClassNames } from '@fluentui/react-utilities';
-import { makeStyles, mergeClasses } from '@griffel/react';
+import { makeStyles, mergeClasses, shorthands } from '@griffel/react';
 import type { OptionGroupSlots, OptionGroupState } from './OptionGroup.types';
 
-/**
- * @deprecated Use `optionGroupClassNames.root` instead.
- */
-export const optionGroupClassName = 'fui-OptionGroup';
 export const optionGroupClassNames: SlotClassNames<OptionGroupSlots> = {
   root: 'fui-OptionGroup',
   label: 'fui-OptionGroup__label',
 };
 
+// TODO(sharing) use theme values once available
+const horizontalSpacing = {
+  xxs: '2px',
+  xs: '4px',
+  sNudge: '6px',
+  s: '8px',
+};
+
 /**
  * Styles for the root slot
  */
 const useStyles = makeStyles({
   root: {
-    // TODO Add default styles for the root element
+    display: 'flex',
+    flexDirection: 'column',
+    rowGap: horizontalSpacing.xxs,
+
+    '&:not(:last-child)::after': {
+      content: '""',
+      ...shorthands.borderBottom(tokens.strokeWidthThin, 'solid', tokens.colorNeutralStroke2),
+      display: 'block',
+      paddingBottom: horizontalSpacing.xs,
+      marginBottom: horizontalSpacing.xs,
+    },
   },
 
-  label: {},
+  label: {
+    ...shorthands.borderRadius(tokens.borderRadiusMedium),
+    color: tokens.colorNeutralForeground3,
+    columnGap: horizontalSpacing.xxs,
```

Comment:
Is this property necessary? I don't think it has any effect here.

Reply:
Good point, I can remove it. It was specified in the design spec, but I don't think there's any actual reason it's needed.

Reply:
Yeah, since this is `display: 'block'` we should probably remove it.